BasketTest.md

```diff
@@ -111,6 +122,5 @@ When submitting the task please also provide an estimate of how long you feel yo
 You can submit the test in the following ways:
 - From a shared link such as drop box, one drive, google drive etc. (preferred)
```

Comment:
Is this still our preferred? I often find `git clone` easier than a zip

Reply:
True, no hard with people seeing other peoples versions.